src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.2.0
+
+### New Feature
+
+- Support for self-signed and private CA with Docker
```

Comment:
```suggestion
- Support for self-signed certificates and private CA with Docker
```

Reply:
What does the "with docker" part refers to?